src/scss/_table.scss

```diff
@@ -1,22 +1,85 @@
+.o-registry-ui__components {
+	padding: 20px;
+	flex-grow: 1;
+}
+
 .o-registry-ui__table-head {
-	background: $_o-registry-ui-base;
+	background-color: oColorsGetPaletteColor('grey-10');
 }
 
 .o-registry-ui__table-body {
-	tr:hover {
-		background: $_o-registry-ui-hover;
+	tr:not(.o-registry-ui__group-title):hover {
+		background-color: oColorsGetPaletteColor('grey-20');;
+	}
+
+	td {
+		vertical-align: middle;
+	}
+
+	.o-registry-ui__group-title {
+		border-bottom: 1px solid oColorsGetPaletteColor('grey-20');;
+
+		td {
+			@include oTypographySansBold(0);
+			@include oTypographyPadding($top: 4, $bottom: 4);
+		}
 	}
 }
 
-.o-registry-ui__table-cell--link {
-	@include oTypographySans($scale: 1);
-	text-decoration: none;
-	color: $_o-registry-ui-text;
+.o-registry-ui__table-cell {
+	&--link {
+		@include oTypographySansBold($scale: 1);
+		text-decoration: none;
+		color: oColorsGetPaletteColor('grey-80');;
+	}
+
+	&--description {
+		@include oTypographySans($scale: 0);
+		@include oTypographyMargin($top: 2, $bottom: 1);
+		color: oColorsGetPaletteColor('grey-60');;
+		font-weight: 400;
+	}
+
+	&--show {
+		display: none;
+		@include oGridRespondTo(M) {
+			display: table-cell;
+		}
+	}
+
+	&--info {
+		@include oTypographyPadding($top: 2);
+		display: table-cell;
+		color: oColorsGetPaletteColor('grey-80');;
+		@include oGridRespondTo(M) {
+			display: none;
+		}
+	};
 }
 
-.o-registry-ui__table-cell--description {
-	@include oTypographySans($scale: 0);
-	@include oTypographyMargin($top: 2);
-	color: $_o-registry-ui-sub-text;
-	font-weight: 400;
+.o-registry-ui__table-cell-label {
+	@include oTypographySansBold(-1);
+	padding: 2px 6px 4px 6px;
+	text-align: center;
+	border-radius: 5px;
+	margin: 0 4px;
+
+	@include oGridRespondTo(M) {
+		margin: 0;
+	};
+
+ 	&--active {
```

Comment:
Could we add a second class to this, `&--maintained`? We're intending on setting the status to this for some of the components that we no longer actively develop but do add bug-fixes. I think it's fine for maintained components to be green too